extensions/amp-mustache/0.2/amp-mustache.js

```diff
@@ -53,13 +53,18 @@ export class AmpMustache extends AMP.BaseTemplate {
     /** @private @const {!JsonObject} */
     this.nestedTemplates_ = dict();
 
-    const content = templateContentClone(this.element);
-    this.processNestedTemplates_(content);
-    const container = this.element.ownerDocument.createElement('div');
-    container.appendChild(content);
+    /** @private {string} */
+    this.template_ = '';
+    if (this.element.tagName == 'TEMPLATE') {
```

Comment:
Needs the same `initTemplateString_` code.

Reply:
opps. thanks for catching.